app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="text.main.style" parent="TextAppearance.AppCompat.Display1">
```

Comment:
your own styles should be in camelCase like:
TextMainStyle

Reply:
Done